pkg/utils/utils.go

```diff
@@ -474,6 +474,15 @@ func GetPfEswitchMode(pciAddr string) (string, error) {
 	return devLinkDeviceAttrs.Mode, nil
 }
 
+// HasRdmaParam returns true if device has "enable_rdma" param
+func HasRdmaParam(bus, pciAddr string) (bool, error) {
```

Comment:
nit: pciAddr -> device
also add some info in comment about expected format of the device depending on the bus

Reply:
Done